
This patch adds module aliases for matching the old dm-raid45 patch
by heinzm@redhat.com carried by kernel-tmb series in order to make
sure there is an upgrade path.

Signed-off-by: Thomas Backlund <tmb@mageia.org>

 drivers/md/dm-raid.c |    2 ++
 1 file changed, 2 insertions(+)

--- linux/drivers/md/dm-raid.c.orig
+++ linux/drivers/md/dm-raid.c
@@ -3689,6 +3689,8 @@ MODULE_ALIAS("dm-raid10");
 MODULE_ALIAS("dm-raid4");
 MODULE_ALIAS("dm-raid5");
 MODULE_ALIAS("dm-raid6");
+MODULE_ALIAS("dm-raid45");
+MODULE_ALIAS("dm-raid4-5");
 MODULE_AUTHOR("Neil Brown <dm-devel@redhat.com>");
 MODULE_AUTHOR("Heinz Mauelshagen <dm-devel@redhat.com>");
 MODULE_LICENSE("GPL");
